scripts/autotracking/location_mapping.lua

```diff
@@ -55,19 +55,19 @@ LOCATION_MAPPING = {
     [1573187]   =   {"@Lightworld/Desert Ledge/Ledge Item"},
     [59791]     =   {"@DP/Desert Palace/Dungeon Chest","@Desert Palace/Big Chest/Big Chest","@Desert Palace/Big Chest/Big Chest"},
     [1573216]   =   {"@DP/Desert Palace/Dungeon Chest","@Desert Palace/Torch/Torch"},
-    [59830]     =   {"@DP/Desert Palace/Dungeon Chest","@Desert Palace/Map Chest/Map Chest","@Desert Palace/vanilla Map Chest/Map Chest"},

```

Comment:
since the hole duplicates with the vanilla items get removed, the entire string should be removed instead of just the "vanilla" part